docs/core/diagnostics/index.md

```diff
@@ -21,7 +21,9 @@ This article helps you find the various tools you need.
 
 ## Metrics
 
-[EventCounters](event-counters.md) allows you to write metrics to identify and monitor performance issues. Metrics incur lower performance overhead compared to tracing, making it more suitable for an always-on performance monitoring. The .NET runtime and libraries publish several [well-known EventCounters](available-counters.md) that you can monitor as well.
+[Metrics](metrics.md) are numerical measurements recorded by apps and libraries over time, often to monitor application health and generate alerts when potential
+problems are detected. Metrics incur lower performance overhead compared to tracing, making them more suitable for an always-on monitoring.
```

Comment:
Minor: What does "tracing" mean here? I wonder if folks would mix "tracing" with "logs with `dotnet-trace`"?

Reply:
yeah the terminology is ambiguous because people use 'tracing' both as shorthand for 'distributed tracing' and
to mean high verbosity logging. I rephrased the message to avoid refering to logging/tracing at all. I think the most
important information is that metrics are cheap and intended for always-on use. Their cost relative to other types of
telemetry feels more nuanced and probably wasn't necessary for the overview text. Let me know if this sounds better.

(There is probably still ambiguity in other parts of the overview doc, like the Logging and Tracing section immediately above, but scoping my changes to just the metrics portion of the text for now)